apps/agora/api/.eslintrc.json

```diff
@@ -12,7 +12,9 @@
     },
     {
       "files": ["*.js", "*.jsx"],
-      "rules": {}
+      "rules": {
+        "tailwindcss/no-custom-classname": "off"
```

Comment:
Added rule as there was a linting error in webpack.config.js

Reply:
We could ignore `webpack.config.js` with:

```
"files": ["*.js", "*.jsx", "!webpack.config.js"],
```

Reply:
We should either ignore `webpack.config.js` or disable the rule just for `webpack.config.js`. The former is either. Note that `webpack.config.js` may be removed in the near future when we switch to esbuild.

Reply:
sounds good!